src/nerve/server/proxy_manager.py

```
@@ -0,0 +1,423 @@
+from typing import TYPE_CHECKING, Any, Literal
+
+if TYPE_CHECKING:
+    from pathlib import Path
+
+logger = logging.getLogger(__name__)
+
+
+class ProxyStartError(Exception):
+    """Failed to start proxy server."""
+
+    pass
```

Ruff (0.14.10)

Unnecessary `pass` statement

Remove unnecessary `pass`

(PIE790)